components/phone_com/app/phone_com.app.ts

```diff
@@ -1,13 +1,71 @@
 import { defineApp } from "@pipedream/types";
+import axios from "axios";
+import { httpRequestParams, apiResponse, sendMessageParams } from "../common/types";
 
 export default defineApp({
   type: "app",
   app: "phone_com",
-  propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _baseUrl(): string {
+      return 'https://api.phone.com/v4';
     },
+    async _httpRequest({ method, endpoint, data }: httpRequestParams): apiResponse {
+      return axios({
+        method,
+        url: this._baseUrl() + endpoint,
+        headers: {
+          Authorization: `Bearer ${this.$auth.access_token}`,
+        },
+        data
+      })
+    },
+    async listAccounts(): apiResponse {
+      return this._httpRequest({
+        method: "GET",
+        endpoint: "/accounts"
+      });
+    },
+    async sendMessage(data: sendMessageParams): apiResponse {
+      return this._httpRequest({
+        method: 'POST',
+        endpoint: `accounts/${this.account}/messages`,
+        data
+      })
+    }
```

Comment:
We should pass `$` and `account` into the `sendMessage` method.
```suggestion
    async sendMessage($, account, data: sendMessageParams): apiResponse {
      return this._httpRequest({
        method: 'POST',
        endpoint: `accounts/${account}/messages`,
        data
      }, $);
    }
```